CONTRIBUTING.md

```diff
@@ -149,8 +133,7 @@ To enable us to quickly review and accept your pull requests, always create one
   - `.html` file has all possible states of the component.
   - `.js` file has scenarios covering these cases.
   - All existing and new tests should pass.
-- Updated changelog.
 
-After verifying that your changes are ready, you can [create a pull request from your fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork). Make sure that an issue is linked to the pull request and that you have a proper description with screenshots.
+After verifying that your changes are ready, you can [create a pull request from your fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork). Make sure that the name of your pull request follows the [Conventional Commits spec](https://www.conventionalcommits.org/), and that you have a proper description with screenshots and a [closing keyword](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests).
```

Comment:
Hm, maybe we need to add this memo to PR template to prevent failing PRs from the start. (at least for me 😃)

Reply:
Oh, this is CSS repo so it doesn't have PR template. Maybe create the same as in React repo?

Reply:
CSS repo never gets external PRs, so I think we'll be fine here.